client/src/components/Q&A/QAwidget.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Search from './Search';
+import QuestionList from './QuestionList';
+
+const QuestionAndAnswer = styled.div`
+ background-color: #F7F6F2;
+`;
+
+const QAwidget = ( props ) => {
+
+  const [questions, setQuestions] = useState([]);
+  const [searchInput, setSearchInput] = useState('');
+
+
+  useEffect(() => {
+    props.getListQuestions({page: 1, count: 5, product_id: props.product_id})
+      .then((response) => {
+        setQuestions(response.results.sort((a, b) => (a.helpness - b.helpness)))
+        // console.log('Here is the questions: ', questions);
+      })
+      .catch(console.log)
+  })
```

Comment:
useEffect needs a second parameter to prevent loops. I'd suggest you use '[]' here.